### ISSUE TITLE
[GIP] Burn Voting

### ISSUE THREAD

**Issue by treeder** [gochain/gip#36]
## Summary

Instead of proving you have X amount of GO to vote with like we've done in the past, voters would vote with GO which would be burned. 

### Pros

* Voters would really have to believe in what they're voting for as it would cost them
* Can't cheat (once you've voted, the GO is gone and can't be moved around to cheat the system)
* Burns GO and reduces supply
* Simple process

### Cons

* Whales would have a clear advantage.... but they already do anyways with the proof based system so 🤷 

## Specification

* Each vote option would have an address, eg: YES address and NO address for yes/no votes. 
* At the cutoff time, the one with the higher balance wins
* After, all tokens in all vote addresses are burnt

## Additional Information

You could also potentially have a **blind vote** so nobody knows what the current tally is. In this case, each voter would choose their answer then get an address to send to. At the cutoff time, all the addresses balances would be summed up then burnt.

**Comment by QuPsi** [comment]
Would this apply to the node validators as well or only community members?

**Comment by treeder** [comment]
Good question. I think it should apply to all including validators, but may be hard to get them to agree to that. Perhaps it could apply to both sides in separate voting pools so it would still be separate votes, but burnt for the validator votes and separately for the community votes.

**Comment by QuPsi** [comment]
Yeah, the only way I'd go for this is if the validators have their Go burned as well. It would be really unfair to the community otherwise. Having the community sacrifice their Go that they _paid_ for while the validators get free Go AND free votes. Definitely not balanced nor fair.

Further, if the concern is users shuffling Go around to vote multiple times from different wallets, could have a locking mechanism where users send their Go to be locked up until the vote is over and then returned afterwards. This is just a thought, not fully fleshed out. Would definitely need more discussion.

I'll be honest, I don't particularly like this idea.

Honestly, I'd completely reverse it. Make it free for community members to vote, but charge the validators Go and burn their fees... they're getting them for free anyway and have amassed quite a substantial amount at this point. I doubt many would go for this, but I don't like the idea of having to sacrifice my investment to vote.

**Comment by QuPsi** [comment]
Just a thought, if the worry is about shuffling Go around... when the votes are tallied, there could be a function to check the amount of Go in a wallet and compare it to the number of votes that was submitted. This way, a user has to maintain their Go in their wallet at the time votes are tallied up. If they submitted 100 votes and there's only 10 Go left if their wallet at the time of vote counting, only 10 votes count.

Just a thought...

**Comment by SAClough** [comment]
Could this result in richer people, who can afford to burn a substantial amount of Go, pushing votes through that would skew things in their favour so that they benefit at the expense of those who couldn't afford to lose their Go in the longer term?
